.shellcheckrc

```diff
@@ -0,0 +1,31 @@
+# ~/.shellcheckrc
+disable=SC2086 # (info): Double quote to prevent globbing and word splitting.
```

Comment:
Create issue to resolve such items.
Put here comment `until ....`
To make it clear it is not permanent suppression.

Reply:
Changes Done

Reply:
@romani IF we will use Terminal then we can only check single file errors. 
For eg: `shellcheck common.sh` We can only see the errors of common.sh

We cant see all the errors of a particular shellcheck code using the terminal.